test/acceptance/features/workloadResourceMapping.feature

```diff
@@ -779,3 +782,304 @@ Feature: Bind services to workloads based on workload resource mapping
                       - name: .metadata.name
             """
         Then Error message is thrown
+
+    @spec
+    Scenario: Altering a workload mapping should project values where there were previously none
```

Comment:
```suggestion
    Scenario: SPEC Altering a workload mapping should project values where there were previously none
```

Please do not use duplicate scenario names - it causes conflicts in tools processing the results. The best practice here is that scenario names must be unique.

Reply:
Fixed in b370078b.